content/questions/foundations/question-109.md

```diff
@@ -0,0 +1,16 @@
+---
+archetype: "questions"
+title: "Question 109"
+question: "What are the different forms of two-factor or multi-factor authentication supported by GitHub? (Choose four.)"
+draft: false
+---
+
+> https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication
+- [x] Passkey
+- [x] Text message
+- [ ] Phone call
+> While text messages are supported in some countries, phone calls are not
+- [ ] Email
+- [x] Security key
+- [x] GitHub mobile
+- [ ] Time-based one-time password (TOTP)
```

Comment:
Isn't this correct?

```suggestion
- [x] Time-based one-time password (TOTP)
```

Reply:
That is on me. I didn't make it clear that TOTP should be included AND marked as correct.